openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -232,16 +232,23 @@
 - control_key: AC-2 (11)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - text: |-
-        This control will only be partially satisfied when OpenShift is
-        configured to connect to an Identity Provider (IdP).
-
-        More work is being done to address this control. Engineering
-        progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-210
+       This functionality would fall to third
+       party software such as an Identity Provider (IdP),
```

Comment:
```suggestion
       party software such as Red Hat Identity Management (Red Hat IdM),
```

Reply:
The nuance here is "Identity Provider (IdP)" being a category, whereas Red Hat IdM, MSAD, LDAP, etc are examples of appropriate 3rd party software.